src/ResourceManager/HDInsight/Commands.HDInsight.Test/UnitTests/ConfigurationTests.cs

```diff
@@ -41,6 +42,43 @@ public void CanCreateNewConfigForRServer()
             CreateNewConfig(setEdgeNodeVmSize: true);
         }
 
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void CanAddSparkCustomConfigs()
+        {
+            AzureHDInsightConfig config = new AzureHDInsightConfig();
+
+            var addConfigValuesCmdlet = new AddAzureHDInsightConfigValuesCommand
```

Comment:
Not really required, it still tests the whether the configs are correctly set. But I've split it into two.